### DISCUSSION TITLE
Option to disable checkerboard background behind transparent images.

### DISCUSSION THREAD

**Discussion by Auratelience** [notaapp/talk#300] [Feature Requests]
Hi Nota team, this isn't a big deal if it can't be done, but it would be nice to have an option to disable the checker-boarding for transparent images. Just having the standard dark grey / white would be nice.

Thanks, Auratelience

**Comment by astoilkov** [comment]
Yes, we could do that. Thanks! (pinging @erusev so he can also read the issue.)
